blockchain/state_manager/src/lib.rs

```diff
@@ -332,9 +332,8 @@ where
     {
         let db = self.blockstore_cloned();
         let lb_wrapper = SMLookbackWrapper {
-            sm: self,
-            store: self.blockstore(),
-            tipset,
+            sm: self.clone(),
```

Comment:
how costly is this clone?

Reply:
It's an `Arc` so it's essentially free.

Reply:
`self` here is an instance of `StateManager` which has a quite bit of fields, a cache included, right? Or am I missing something?

Reply:
Ah no, it's a funny self, my bad.

Reply:
Yep, a funny self. :)

Reply:
Maybe I should change it to `Arc::clone(self)` just to be more explicit.

Reply:
I guess it wouldn't hurt and it would make this piece of code less surprising. I'm good with both approaches.